stdlib/2and3/builtins.pyi

```diff
@@ -708,6 +714,9 @@ class bytearray(MutableSequence[int], ByteString):
     def lower(self) -> bytearray: ...
     def lstrip(self, __bytes: Optional[bytes] = ...) -> bytearray: ...
     def partition(self, __sep: bytes) -> Tuple[bytearray, bytearray, bytearray]: ...
+    if sys.version_info >= (3, 9):
+        def removeprefix(self, __prefix: bytearray) -> bytearray: ...
```

Comment:
The argument type should actually be bytes. Typecheckers assume that bytearray is a subclass of bytes.

```
In [12]: ba = bytearray(b'xyz')                                                                                                                                                      

In [13]: ba.removeprefix(b'x')                                                                                                                                                       
Out[13]: bytearray(b'yz')

In [14]: b'xyz'.removeprefix(bytearray(b'x'))                                                                                                                                        
Out[14]: b'yz'
```